.github/workflows/publish-community-dashboard.yml

```diff
@@ -0,0 +1,37 @@
+name: Build-Push kotal dashboard
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    
+    - name: Login to DockerHub
+      uses: docker/login-action@v2
+      with:
+        registry: registry.digitalocean.com
+        username: ${{ secrets.DIGITAL_OCEAN_CONTAINER_REGISTRY_USERNAME }}
+        password: ${{ secrets.DIGITAL_OCEAN_CONTAINER_REGISTRY_PASSWORD }}
```

Comment:
use hardcoded `kotalco` for username, and use `DOCKERHUB_PASSWORD` secret for password.